templates/certified/search-results.html

```diff
@@ -55,9 +55,9 @@ <h1 class="u-sv3">Ubuntu certified
                </li>
                <li class="p-accordion__group">
                  <div role="heading" aria-level="2" class="p-accordion__heading">
-                   <button type="button" class="p-accordion__tab" id="tab2" aria-controls="tab2-section" aria-expanded="false">Certified Ubuntu release</button>
+                   <button type="button" class="p-accordion__tab" id="tab2" aria-controls="tab2-section" aria-expanded="true">Certified Ubuntu release</button>
                  </div>
-                 <section class="p-accordion__panel" id="tab2-section" aria-hidden="true" style="padding-left: 1rem;" aria-labelledby="tab2">
+                 <section class="p-accordion__panel" id="tab2-section" aria-hidden="false" style="padding-left: 1rem;" aria-labelledby="tab2">
```

Comment:
Can you we not have this padding-left in the .scss?

Reply:
I see that this is done in a few places throughout u.c - I think it's OK to have this small amount of inline style for nudges like this, however, because it's a repeated pattern, I also think it'd be worth suggesting an amendment to the accordion pattern in Vanilla, to introduce something like an `is-dense` modifier that reduces the padding the way this is doing. It might not get accepted, but we should have the conversation.

Reply:
Ok sure, I'll make an issue to suggest the amendment